### ISSUE TITLE
MPV Occasionally Freezing When Playing RTSP Stream

### ISSUE THREAD

**Issue by jsdobkin** [mpv-player/mpv#5039]
### mpv version and platform
MPV 0.27.0
Windows 10, Version 1607 (OS Build 14393.15)

### Reproduction steps
`cmd /c C:\Users\jsdobkin\Downloads\mpv-x86_64-20170913\mpv rtsp://<ip_address>:<port>/test_case_1.1 --rtsp-transport=udp --geometry=192x108+0+0 --no-border --msg-level=all=v --force-window=immediate --priority=realtime`

Note that this command will be executed anywhere between 1 to N times (where N is usually no more than 40) in parallel. Also, flags after '--no-border' were inserted for debugging purposes, but the issue is still present without them.

### Expected behavior
Stream begins to play and stops appropriately.

### Actual behavior
Stream launches terminal, does some initial setup, then freezes before playing any content. CPlayer does not launch in this case if '--force-window=immediate' is not present.

### Log file
Unfortunately it is hard to manage log files given I am launching several streams in parallel and launching hundreds per hour, so I have taken a screenshot of exactly when the issue occurs (in verbose mode) and does not perform any additional work until killed. When this issue occurs is always lands in the same exact spot here `[lavf] Found 'rtsp' at score=100 size=0 (forced).` When the stream IS successful, it begins to print the SDP details immediately after this line.
![image](https://user-images.githubusercontent.com/33096091/32005706-d090cbe0-b959-11e7-96ad-a7d1af58f80e.png)

### Sample files
I am unable to provide sample files for what I am actually testing as the are proprietary Intel media clips for internal use only, but I do not think this issue is content sensitive since I use a gauntlet of different source-clips (MPEG-2, H264, H265 codecs encapsulated in MPEG-TS with various resolutions, and bit-rates -- no audio) and even run them through a Real-Time transcoder in a lot of cases.

**Comment by jsdobkin** [comment]
[mpv-freeze.log](https://github.com/mpv-player/mpv/files/1415270/mpv-freeze.log)

I was fortunately able to grab the complete contents of one of the verbose logs from terminal

**Comment by wm4** [comment]
Most likely a ffmpeg issue. I don't think there was any work done on RTSP recently, though.

**Comment by rgaufman** [comment]
Any solution to this? - live555 seem to have a better rtsp implementation, but not sure if it's supported in mpv.

**Comment by Dudemanguy** [comment]
Please open a new issue if this is still a problem on 0.36.
